static/js/containers/pages/login/LoginEmailPage.js

```diff
@@ -60,15 +60,15 @@ export class LoginEmailPage extends React.Component<Props> {
         title={`${SETTINGS.site_name} | ${LOGIN_EMAIL_PAGE_TITLE}`}
       >
         <div className="container auth-page">
-          <div className="row auth-header">
-            <h1 className="col-12">Sign in</h1>
-          </div>
           <div className="row auth-card card-shadow auth-form">
             <div className="col-12">
+              <div className="row auth-header">
+                <h1 className="col-12">Sign in</h1>
+              </div>
```

Comment:
```suggestion
              <h1>Sign in</h1>
```

Sorry I missed this on the first round. This extra div is not necessary. All it appears to be doing is adding some spacing below the `h1`. If we need spacing below the `h1`, we can just define it here with no added class and style it with `.auth-card h1`. This comment applies to all other pages where there is a `<div className="row">` with children that all have `className="col-12"`. I think I'm seeing that this pattern was used in some other parts of the codebase but unless I'm missing some necessary styling property it's adding, we should get rid of it

Reply:
The spacing under the `h1` is actually conditional. Whenever a sub-heading exists for example "Signing in as {username}", the spacing removed. Also pages like `RegisterConfirmSentPage.js` and `LoginForgotPasswordPage.js` do not have a `div` separating heading from the content which stops us from applying the spacing based on number of siblings.